src/integration-test/groovy/pages/modules/OrganisationAdminTab.groovy

```diff
@@ -5,8 +5,29 @@ import geb.Module
 class OrganisationAdminTab extends Module{
 
     static content = {
+
         editButton{$('[data-bind="click:editOrganisation"]')}
-       deleteButton{ $('[data-bind="click:deleteOrganisation"]')}
+        deleteButton{ $('[data-bind="click:deleteOrganisation"]')}
+
+        configOverrideTab(required:false) { $('#configuration-tab')}
```

Comment:
There is no override for an organisation - this should just be the "config" tab instead of "configOverride"